conf/carbon.conf.example

```diff
@@ -177,6 +177,9 @@ LOG_CACHE_QUEUE_SORTS = False
 # there are a large number of metrics which receive very frequent updates OR if
 # disk i/o is very slow.
 #
+# bucketmax (experimental) - As 'max' but uses a different algorithm to determine
+# the metric with the most datapoints. Should perform better than 'max'.
+#
```

Comment:
Maybe you should mention " number of datapoints per metrics vs total number of
unique metrics" from PR description here too.

Reply:
I guess so. Thing is that that statement is an assumption. This whole thing needs to be better tested to see how it behaves in real life and if my assumption is correct.

Reply:
Sure, my idea was to extend that a bit for users, e.g. "Should perform better than 'max' if number of datapoints per metric is relatively low compared to the total number of
unique metrics, which should be common use-case. See https://github.com/graphite-project/carbon/pull/879 for details."
But that's optional ofc.

Reply:
added some more info